src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,49 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private K[] keyItem;
+    private V[] itemPair;
+    private int size;
+
+    public StorageImpl() {
+        keyItem = (K[]) new Object[MAX_STORAGE_SIZE];
+        itemPair = (V[]) new Object[MAX_STORAGE_SIZE];
+    }
+
+    private boolean isKeyTheSame(K key, K[] keyItem, int i) {
+        boolean isKeyNull = (key == null && keyItem[i] == null);
+        boolean isKeyNotNull = (key != null && key.equals(keyItem[i]));
+        return isKeyNull || isKeyNotNull;
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            boolean isKeyTheSame = isKeyTheSame(key, keyItem, i);
+            if (isKeyTheSame) {
+                itemPair[i] = value;
+                size--;
```

Comment:
you don't need to change a size here. 
Just call `return;`
```suggestion
                return;
```